packages/convex-helpers/server/hono.ts

```diff
@@ -104,20 +104,25 @@ export class HttpRouterWithHono<
    * @returns - an array of [path, method, endpoint] tuples.
    */
   override getRoutes = () => {
-    const convexRoutes: [string, RoutableMethod, (...args: any) => any][] = [];
+    const convexRoutes: [string, RoutableMethod, PublicHttpAction][] = [];
 
     // Likely a better way to do this, but hono will have multiple handlers with the same
     // name (i.e. for middleware), so de-duplicate so we don't show multiple routes in the dashboard.
     const seen = new Set();
     this._app.routes.forEach((route) => {
+      // The (internal) field _handler on PublicHttpAction is used to look up the function's line number.
+      const handler = route.handler as any;
+      handler._handler = route.handler;
+      handler.isHttp = true;
```

Comment:
why wouldn't it already be `isHttp`?

Reply:
It might be, but I can't tell from the type. `route.handler` has a Hono type, and as far as i know they might re-wrap the handler.